larray/excel.py

```diff
@@ -453,9 +453,58 @@ def load(self, header=True, convert_float=True, nb_index=None, index_col=None):
                 return LArray(list_data)
 
     # XXX: remove this function?
-    def open_excel(filepath=None, **kwargs):
-        return Workbook(filepath, **kwargs)
+    def open_excel(filepath=None, new_file=False, **kwargs):
+        """
+        Open an Excel workbook.  
+        
+        Parameters
+        ----------
+        filepath: str
+            Path to an Excel file. The file must exist if `new_file` is not set to True.
+        new_file: bool, optional
+            If True and filepath represents a not existing Excel file, creates a new file.
+        \**kwargs
+        
+        Returns
+        -------
+        Excel workbook.
+        
+        Examples
+        --------
+        >>> from larray import *
+        >>> from larray.tests.test_la import abspath
+        >>> arr = ndtest((3, 3))
+        >>> arr
+        a\\b  b0  b1  b2
+         a0   0   1   2
+         a1   3   4   5
+         a2   6   7   8
+        
+        create and save an array
+        
+        >>> # argument new_file of open_excel must be set to True if you want to create a new Excel file
+        >>> wb = open_excel(abspath('excel_file.xlsx'), True)
+        >>> wb['arr'] = arr.dump()
+        >>> wb.save()
+        >>> wb.close()
+        
+        read array from Excel file
+        
+        >>> wb = open_excel(abspath('excel_file.xlsx'))
+        >>> arr2 = wb['arr'].load()
+        >>> arr2
+        a\\b  b0  b1  b2
+         a0   0   1   2
+         a1   3   4   5
+         a2   6   7   8
+        >>> wb.close()
+        """
+        if not os.path.isfile(filepath) and not new_file:
+            raise ValueError("File {} does not exist. Please give the path to an existing file "
+                             "or set argument `new_file` to True to create a new file.".format(filepath))
+        else:
+            return Workbook(filepath, **kwargs)
 else:
-    def open_excel(filepath=None, **kwargs):
+    def open_excel(filepath=None, new_file=False, **kwargs):
```

Comment:
the new_file argument should rather be handled in Workbook. Also I think a mode argument  ("w" | "r" | "a") would be more consistent with the rest of Python. Opening a file read-only is currently not supported by xlwings (https://github.com/ZoomerAnalytics/xlwings/issues/318), so we will need to add a warning in the doc that the file can be modified anyway.

FWIW, using win32com, something like this might work:
```
wb = excel.Workbooks.Open(excelPath, ReadOnly=1)
# OR
wb = excel.Workbooks.Open(excelPath, ReadOnly=1)
```